### PR TITLE
Finishing Power & Battery tab

### PR DESCRIPTION
**Pull request by basdelfos**
Finished the new Power & Battery tab in the configurator.

Needs Betaflight PR: https://github.com/betaflight/betaflight/pull/3532

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -2149,15 +2149,18 @@
     },
 
     "powerAmperageScale": {
-        "message": "Scale the output voltage to milliamps [1/10th mV/A]"
+        "message": "Scale"
     },
     "powerAmperageOffset": {
-        "message": "Offset in millivolt steps"
+        "message": "Offset in milliampere steps"
```
Comment (Cleric-K) [inline _locales/en/messages.json]: Why has this been changed? It seems to be incorrect. The offset really is in millivolts (as measured by the ADC), not milliamperes. Here is the code from current.c
https://github.com/betaflight/betaflight/blob/master/src/main/sensors/current.c#L107-L112

And the scale units [1/10th mV/A] are also nice to display.

Comment (mikeller) [reply _locales/en/messages.json]: @Cleric-K: Yep, that's correct, well spotted. Would you mind opening a pull request to fix this?

**Comment by basdelfos** [comment]
I changed this when refining the power & battery tab.
I thought it was some kind of copy-paste error or typo in the language
resource file. But now looking a the code you linked, I understand why  the
offset is in millivolts. It is the offset of the 3v3 voltage on the pad not
an offset added to the calculated current.


2017-11-01 15:06 GMT+01:00 Michael Keller <notifications@github.com>:

**Comment by Cleric-K** [comment]
@basdelfos Yes, it would probably be more user friendly if the offset was really in milliamperes (added after scaling), because when calibrating, one could first figure out the scale to match the slope and then simply measure the offset. The ADC voltage is not readily visible so one should in most cases set the offset by trial and error. But I guess it is too late to change that in CF/BF.

@mikeller I can do it, but let's first agree on what exactly the labels should be. Should we simply revert them to what they were before this commit?

**Comment by mikeller** [comment]
@Cleric-K: I'd go for `Offset in millivolt steps [1/10th mV/A]`. Agreed that changing the scaling around to do the offset after scaling might be better, but seeing that a lot of the target definitions would have to be changed / retested since they define presets for the scaling, and that all of the videos / blog posts / instructions about calibrating the current sensor would have to be redone, I think leaving the calculation as it is is the better option.

**Pull request merged**